master/reference/calicoctl/setup/etcdv2.md

```diff
@@ -1,132 +1,63 @@
 ---
-title: Configuring calicoctl - etcdv2 datastore 
-layout: docwithnav
+title: Connecting calicoctl to etcd
 ---
 
-This document covers the configuration options for calicoctl when using an etcdv2 datastore.
-
-## Configuration file 
+## Configuration File
 
 By default `calicoctl` looks for a configuration file at `/etc/calico/calicoctl.cfg`.
 
 The file location may be overridden using the `--config` option on commands that required
 datastore access.
 
-The config file is a yaml map, with the following supported fields: 
-
-| Field           | Description                                | Examples
-|-----------------|--------------------------------------------|----------
-| etcdEndpoints   | A comma separated list of etcd endpoints.  | http://etcd1:2379 
-| etcdUsername    | Username for RBAC (optional)               | "user" 
-| etcdPassword    | Password for the given username (optional) | "password"
-| etcdKeyFile     | Path to the etcd key file (optional)       | /etc/calico/key.pem
-| etcdCertFile    | Path to the etcd client cert (optional)    | /etc/calico/cert.pem
-| etcdCACertFile  | Path to the etcd CA file (optional)        | /etc/calico/ca.pem
-
-> **NOTE** 
-> 
-> If you are running with TLS enabled, ensure your endpoint addresses use https
-
-Each of the above configuration options can also be set through environment variables.  For example,
-`etcdEndpoints` can also be represented as `ETCD_ENDPOINTS`.
-
-#### Example configuration file
+The following is an example `calicoctl.cfg`, which lists all supported fields:
 
 ```yaml
-etcdEndpoints: http://etcd1:2379,http://etcd2:2379,http://etcd3:2379 
-etcdKeyFile: /etc/calico/key.pem 
-etcdCertFile: /etc/calico/cert.pem 
-etcdCACertFile: /etc/calico/ca.pem 
-```
-
-#### Example using environment variables 
-
-```
-ETCD_ENDPOINTS=http://myhost1:2379 calicoctl get bgppeers
-```
-
-# Additional information
-
-## Configure role based access for etcd v2
```

Comment:
Yeah, as I said a few places above, I think RBAC should be moved to another guide that's more of a "user walkthrough / tutorial", or just removed and included as a link to the etcd docs.